pom.xml

```diff
@@ -564,17 +569,35 @@
 					</exclusion>
 				</exclusions>
 			</dependency>
+			<dependency>
+				<groupId>org.testcontainers</groupId>
+				<artifactId>testcontainers</artifactId>
+				<version>1.21.3</version>
+			</dependency>
+			<dependency>
+				<groupId>org.testcontainers</groupId>
+				<artifactId>junit-jupiter</artifactId>
+				<version>1.21.3</version>
+			</dependency>
 			<dependency>
 				<groupId>org.testcontainers</groupId>
 				<artifactId>mysql</artifactId>
 				<version>1.21.3</version>
-				<scope>test</scope>
 			</dependency>
 			<dependency>
 				<groupId>org.testcontainers</groupId>
 				<artifactId>postgresql</artifactId>
 				<version>1.21.3</version>
-				<scope>test</scope>
+			</dependency>
+			<dependency>
+				<groupId>software.amazon.awssdk</groupId>
+				<artifactId>s3</artifactId>
+				<version>2.31.50</version>
+			</dependency>
+			<dependency>
+				<groupId>com.adobe.testing</groupId>
```

Comment:
Aren't these libraries only for testing? And hence providing it as the scope?

Reply:
See https://stackoverflow.com/a/20750041